src/commands/EjectCommand.js

```diff
@@ -0,0 +1,47 @@
+import Command from "../Command";
+import FileSystemUtilities from "../FileSystemUtilities";
+import async from "async";
+import path from "path";
+import userHome from "user-home";
+
+export default class EjectCommand extends Command {
+  initialize(callback) {
+    this.to = this.flags.to || userHome;
```

Comment:
Unlikely scenario but incase `userHome` is also null should we handle this inside of initialize and log out an error?